### ISSUE TITLE
Question 

### ISSUE THREAD

**Issue by sadvlad212** [Kingsman44/Pixelify#53]
Just a quick question about Gboard. Is it possible to turn on assistant voice typing. And if it's not why (is it just device specific or a Google thing).
![Screenshot_2022-02-11-18-23-56-792_com google android inputmethod latin](https://user-images.githubusercontent.com/37769947/153639700-abacccf7-195b-462c-bf03-76f2a72db057.jpg)

PS: Amazing work, with the release of Pixelify 2.0, literally all problems that i had are fixed. (Mi 11i on MIUI 12.5.4 A11)

Thanks for answer in advance.

**Comment by Kingsman44** [comment]
Well, it's not like that, but it depends on serveral factor
1) Google apk, Android 10+ apk from apkmirror
https://www.apkmirror.com/apk/google-inc/google-search/google-search-13-5-12-release/google-app-13-5-12-29-arm64-android-apk-download/

2) Google text to speech

3) Google Android system Intelligence

4) Your main language and gboard language to en-US

also i am unsure that it's work on Android 11.
but it works perfectly in Android 12

**Comment by Kingsman44** [comment]
after checking those criteria i recommend to reboot your device and check

**Comment by sadvlad212** [comment]
Yeah it seems that it's a A11 thing. I would say that i checked all the criteria. Thanks for answer tho. I'll get A12 soon so I'll just be patient.
